src/DetailsView/components/tab-stops/tab-stops-test-view-controller.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import {
+    EditExistingFailureInstancePayload,
+    TabStopsViewActions,
+} from 'DetailsView/components/tab-stops/tab-stops-view-actions';
+
+export class TabStopsTestViewController {
```

Comment:
I think the controller is a great way to handle what we chatted about yesterday. I like this and it feels like it will definitely be less confusing and more palatable!